gems/actionpack/CVE-2021-22942.yml

```diff
@@ -0,0 +1,70 @@
+---
+gem: actionpack
+framework: rails
+cve: 2021-22942
+url: https://groups.google.com/g/rubyonrails-security/c/wB5tRn7h36c
+title: Possible Open Redirect in Host Authorization Middleware
+date: 2021-08-19
+description: |
+  There is a possible open redirect vulnerability in the Host Authorization middleware in Action Pack. This vulnerability has been assigned the CVE identifier CVE-2021-22942.
+
+  Versions Affected: >= 6.0.0. Not affected: < 6.0.0 Fixed Versions: 6.1.4.1, 6.0.4.1
+
+  Impact
+  ------
+
+  Specially crafted “X-Forwarded-Host” headers in combination with certain “allowed host” formats can cause the Host Authorization middleware in Action Pack to redirect users to a malicious website.
+
+  Impacted applications will have allowed hosts with a leading dot. For example, configuration files that look like this:
+
+  ```ruby
+  config.hosts <<  '.EXAMPLE.com'
+  ```
+
+  When an allowed host contains a leading dot, a specially crafted Host header can be used to redirect to a malicious website.
+
+  This vulnerability is similar to CVE-2021-22881, but CVE-2021-22881 did not take in to account domain name case sensitivity.
+  Releases
```

Comment:
Needs newline before it and dotted lines beneath it like the other section headers.

Reply:
Fixed